core/server/helpers/img_url.js

```diff
@@ -0,0 +1,14 @@
+// # Basically the same as {{content words="0"}} Content Helper
+// This is a temporary hack to enable lts themes to be forward compatible
+
+var hbs             = require('express-hbs'),
+    downzero        = require('../utils/downzero'),
+    content;
+
+content = function () {
+    return new hbs.handlebars.SafeString(
+        downzero(this.html)
```

Comment:
This behaviour is very different to what  `{{img_url}}` does. IMO it should be possible to copy the helper from master and have it work with a couple of minor adjustments:

- don't use proxy, include url util directly
- ensure error handling works with LTS (can't remember if the format is the same).

Maybe you already tried this and it didn't work for some reason I can't see?